example/Lynx/Page/Form.purs

```diff
@@ -108,11 +107,11 @@ component =
         , fragment: appendFragment profile1
         }
 
-eval
-  :: ∀ m
-   . MonadAff m
-  => Query
-  ~> H.ParentDSL State Query ChildQuery ChildSlot Message m
+eval ::
+  ∀ m.
+  MonadAff m =>
+  Query
+    ~> H.ParentDSL State Query ChildQuery ChildSlot Message m
```

Comment:
Although, I must add, everywhere else we put our punctuation at the start of a newline, instead of ending our line with it.
```purescript
-- no
val = [
  "foo",
  "bar",
  "baz"
]

-- yes
val =
  [ "foo"
  , "bar"
  , "baz"
  ]

-- no 
val2 =
  wrap $
    first <>
    second <>
    third

-- yes 
val2 =
  wrap
    $ first
    <> second
    <> third

-- no
type MyType =
  FooBar \/
  Biz \/
  Bazel

-- yes
type MyType
  = FooBar
    \/ Biz
    \/ Bazel

-- yes?
myFunc ::
  forall a.
  Show a =>
  a ->
  String

-- no?
myFunc
  :: forall a
   . Show a
  => a
  -> String
```

Reply:
> Is there nothing that can be done about this `~>` though? Shouldn't this be:
> 
> ```purescript
> eval ::
>   ∀ m.
>   MonadAff m =>
>   Query ~>
>   H.ParentDSL State Query ChildQuery ChildSlot Message m
> ```

`(~>)` is a [user-land operator][nattrans]. All user-land operators are formatted with the same rules: if the entire thing spans multiple lines, format the left operand, indent the operator, then format the right operand; otherwise, format the whole thing over a single line. We (CitizenNet) can put it on a single line if that's more appealing.

I'm not against a different formatting for user-land operators if it leads to less noisy diffs and doesn't special case anything in the user-land. But, it'll be applied uniformly to all user-land operators. Feel free to [open an issue][new issue]

[nattrans]: https://pursuit.purescript.org/packages/purescript-prelude/4.1.1/docs/Data.NaturalTransformation#t:type%20(~>)
[new issue]: https://gitlab.com/joneshf/purty/issues/new